bzt/modules/python.py

```diff
@@ -679,7 +679,7 @@ def gen_action(self, action_config, indent=None):
                     action_elements.append(self.gen_statement(
                         tpl % (bys[tag], selector, "clear()"), indent=indent))
                 action = "send_keys(_tpl.apply(%r))" % str(param)
-                if isinstance(param, str) and param.startswith("KEY_"):
+                if isinstance(param, (string_types, text_type)) and param.startswith("KEY_"):
```

Comment:
If I understood correctly only difference is py2+unicode case. Please add test to illustrate it.

Reply:
Perfect.
https://github.com/Blazemeter/taurus/pull/851/commits/0359f40c677f99a585b1f886429b41107b31c35a
and
https://github.com/Blazemeter/taurus/pull/851/commits/0cdbccb6c7086a40576328d4140e1b6c04589f5b
When the problem happens, it is easily detected because "KEY_ENTER" is sent as keystokes, which is an error, it is now verified by the test. Tested already in py2/3